assets/stories/skate-components/routePill.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { RoutePill } from "../../src/components/routePill"
+
+const meta = {
+  component: RoutePill,
+} satisfies Meta<typeof RoutePill>
+
+export default meta
+type Story = StoryObj<typeof RoutePill>
+
+export const Default: Story = {
+  args: { routeName: "66" },
+}
+
+export const BlueLine: Story = {
+  args: { routeName: "Blue Line" },
+}
+
+export const LargeFormat: Story = {
+  args: { routeName: "66", className: "c-route-pill--large-format" },
```

Comment:
Nit (non-blocking): I wonder if this would be easier to use if we had a boolean flag on the component itself, so that you would render it as

```tsx
<RoutePill />
```

Or

```tsx
<RoutePill largeFormat />
```

And then move the CSS-ifying into the component itself.

Reply:
That's fine! Then if we ever reuse the large format, the className to be used will be enforced by the component.